crates/orderbook-commons/src/lib.rs

```diff
@@ -149,7 +149,7 @@ pub enum Message {
         order: Order,
         filled_with: FilledWith,
     },
-    Rollover,
+    Rollover(Option<String>),
```

Comment:
Why don't you use the proper type for contract_id?

Reply:
Because the type `ContractId` is not known in the `orderbook_commons` and I prefer sending a string over a byte array :)